lib/commands/file.js

```diff
@@ -65,6 +66,7 @@ options(program, 'files-upload <file> <datasetID>')
   .option('--parallel <parallel>', 'For multipart uploads, the maximum number of concurrent uploads to do at one time.')
   .option('--recursive', 'If <file> is a directory, uploads all files in the directory and any sub directories.')
   .option('--overwrite', 'Overwrite any existing files in the dataset.')
```

Comment:
The `--overwrite` option allows one to tell file-service to always overwrite existing files.   It would seem like a user could use this option if they wish to ignore the already exists errors, right?

Reply:
Correct - just verified.  Using the `--overwrite` option will upload the files, replacing any that already exist with the same name.  Put another way, ignoring "already exists" errors only applies when not using the `--overwrite` option.